db/schema.rb

```diff
@@ -20,13 +20,16 @@
     t.datetime "updated_at", null: false
   end
 
+  add_index "microposts", ["user_id", "created_at"], name: "index_microposts_on_user_id_and_created_at"
+
   create_table "users", force: :cascade do |t|
     t.string   "name"
     t.string   "email"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",                      null: false
```

Comment:
[Q] Why do you have extra spaces?

Reply:
actually, the rake-migrate formats itself. i do not do any thing except to run the command.